### DISCUSSION TITLE
Ender 3  s1 start code??

### DISCUSSION THREAD

**Discussion by ctsvnino** [mriscoc/Ender3V2S1#94] [General]
Will g29 work now before every print? Or it will
Not? So I know it didn't work because it needed to
Cool before probing then heat up
Then cool to
Probe again.

**Comment by mriscoc** [comment]
You can put G29 after G28 if you need to build a new mesh before printing. It is not necessary to cool the bed.

**Reply by ctsvnino** [reply]
It did it itself. It was a thing I read s1 did. Firmware issue

**Reply by mriscoc** [reply]
Please elaborate the exact procedure to replicate and take a video if it is possible. I didn't receive any report about automatic cooldowns. Maybe there is a problem with the start G-code script.
